app/src/main/java/org/sopt/dosopttemplate/ui/doandroid/DoAndroidFragment.kt

```diff
@@ -0,0 +1,32 @@
+package org.sopt.dosopttemplate.ui.doandroid
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import org.sopt.dosopttemplate.databinding.FragmentDoAndroidBinding
+
+class DoAndroidFragment : Fragment() {
+    private var _binding: FragmentDoAndroidBinding? = null
+    private val binding: FragmentDoAndroidBinding
+        get() = requireNotNull(_binding)
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?,
+    ): View? {
+        _binding = FragmentDoAndroidBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+    }
+
+    override fun onDestroyView() {
+        super.onDestroyView()
+        _binding = null
```

Comment:
```suggestion
        _binding = null
        super.onDestroyView()
```
이렇게 _binding=null을 먼저 처리해주면 더 안정성 있게 메모리를 해제할 수 있다고 합니다.

Reply:
오호 이렇게 수정해야겠네요! 좋은 정보 감솸다~